include/cmor.h

```diff
@@ -2,8 +2,8 @@
 #define CMOR_H
 
 #define CMOR_VERSION_MAJOR 3
-#define CMOR_VERSION_MINOR 6
-#define CMOR_VERSION_PATCH 1
+#define CMOR_VERSION_MINOR 7
+#define CMOR_VERSION_PATCH 0
 
 #define CMOR_CF_VERSION_MAJOR 1
 #define CMOR_CF_VERSION_MINOR 6
```

Comment:
obviously linked to the CF version defined in the `Src/cmor_fortran_interface.f90` as well